modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -1,10 +1,11 @@
-import { Arr, Obj, Optional, Optionals } from '@ephox/katamari';
+import { Arr, Obj, Optional, Optionals, Type } from '@ephox/katamari';
 import { Css, PredicateFilter, SugarElement, SugarNode } from '@ephox/sugar';
 
 import DOMUtils from '../api/dom/DOMUtils';
 import DomTreeWalker from '../api/dom/TreeWalker';
 import Editor from '../api/Editor';
 import * as Options from '../api/Options';
+import { Tools } from '../api/PublicApi';
 import { EditorEvent } from '../api/util/EventDispatcher';
```

Comment:
```suggestion
import { EditorEvent } from '../api/util/EventDispatcher';
import Tools from '../api/util/Tools';
```
This was importing from the wrong place. We really should look at adding a lint rule for this.